pkg/resource/firewall/hook.go

```diff
@@ -27,3 +36,251 @@ var (
 		ackrequeue.DefaultRequeueAfterDuration,
 	)
 )
+
+const ErrSyncingPutProperty = "Error syncing property '%s'"
+
+// createLoggingConfig creates logging config for a firewall.
+func (rm *resourceManager) createLoggingConfig(
+	ctx context.Context,
+	r *resource,
+) error {
+	if r.ko.Spec.LoggingConfiguration != nil {
```

Comment:
Can we add a
```
rlog := ackrtlog.FromContext(ctx)
exit := rlog.Trace("rm.createLoggingConfig")
defer exit(err)
```
same thing for delete function

Reply:
done